common.sh

```diff
@@ -35,9 +35,30 @@ delete_pg()
     if [ ${ENABLE_PLACEMENT_GROUP} -eq 0 ] || [ -z $PLACEMENT_GROUP ]; then
         return 0
     fi
-    AWS_DEFAULT_REGION=us-west-2 aws ec2 delete-placement-group \
-        --group-name ${PLACEMENT_GROUP}
-    return $?
+    local ret=0
+    local retry=20
+    while [ "$retry" -ge 0 ]; do
+        echo "Start deleting placement group ${PLACEMENT_GROUP}."
```

Comment:
Why have this print in the loop?  We don't need to see this print 20 times a run.  Put it outside the loop, and then have a "done" or "failed' message at the end.

Reply:
Fixed in latest revision.